lib/gcloud/datastore/commit.rb

```diff
@@ -51,10 +52,47 @@ def initialize
       #   end
       #
       def save *entities
-        entities.each { |entity| shared_upserts << entity }
+        entities.each { |entity| @shared_upserts << entity }
         # Do not save yet
         entities
       end
+      alias_method :upsert, :save
+
+      ##
+      # Inserts entities to the Datastore.
+      #
+      # @param [Entity] entities One or more Entity objects to insert.
+      #
+      # @example
+      #   gcloud = Gcloud.new
+      #   dataset = gcloud.datastore
+      #   dataset.commit do |c|
+      #     c.insert task1, task2
+      #   end
+      #
+      def insert *entities
+        entities.each { |entity| @shared_inserts << entity }
+        # Do not insert yet
+        entities
+      end
+
+      ##
+      # Updates entities to the Datastore.
+      #
+      # @param [Entity] entities One or more Entity objects to update.
+      #
+      # @example
+      #   gcloud = Gcloud.new
+      #   dataset = gcloud.datastore
+      #   dataset.commit do |c|
+      #     c.insert task1, task2
```

Comment:
(The code example for `Commit#update` was showing `insert` instead of `update`, but this comment was made on `Commit#insert`, which was already correct.)